tests/nightly/broken_link_checker_test/test_broken_links.py

```diff
@@ -31,6 +31,7 @@ def prepare_link_test_result(command_output):
     # Whitelisted broken links patterns
     HTTP_403_REGEX = "(HTTP_403)"
     HTTP_401_REGEX = "(HTTP_401)"
+    HTTP_409_REGEX = "(HTTP_409)"
```

Comment:
I think we need 301 and 303, as those are common redirect responses.
Or just do 3xx if you can so it ignores any kind of redirection.